content/en/docs/tasks/debug-application-cluster/audit.md

```diff
@@ -419,6 +419,97 @@ Note that in addition to file output plugin, logstash has a variety of outputs t
 let users route data where they want. For example, users can emit audit events to elasticsearch
 plugin which supports full-text search and analytics.
 
+## Webhook Collector Examples
+
+### Use Falco to collect audit events
+
+[Falco](falco_website) is an open source project for intrusion and abnormality detection for Cloud Native platforms.
+In this example, we will see how to send audit events to the Kubernetes Audit endpoint exposed by Falco.
```

Comment:
```suggestion
In this example, you will see how to send audit events to the Kubernetes Audit endpoint exposed by Falco.
```
(per [address the reader as ”you”](https://kubernetes.io/docs/contribute/style/style-guide/#address-the-reader-as-you) from the documentation style guide).